Extensions/P2P/JsExtension.js

```diff
@@ -296,6 +296,22 @@ module.exports = {
       .addIncludeFile('Extensions/P2P/B_p2ptools.js')
       .setFunctionName('gdjs.evtTools.p2p.getEventData');
 
+    extension
+      .addStrExpression(
+        'GetEventSender',
```

Comment:
I unfortunately missed this for `GetEventData` too, but expressions almost never have `Get` in their name, for conciseness and because it's implied all expressions are getter in GDevelop.

Reply:
Ah I see, I'll keep this in mind in the future